website/docs/addon_applications_admin.md

```diff
@@ -1,26 +1,230 @@
+::: -->
 
-![settings_applications](assets/settings/applications_01.png)
+![settings_applications](assets/applications/admin_applications_layers.png)
 
-### Environments
+The Addon features two primary settings categories: Definitions and Filters:
+
+1. **Definitions:** available only in studio settings, allows to define all possible applications and tools that can be used across all projects.
+2. **Filters:** defines when certain applications and tools are used on project level. This filters whether a certain application or tool should be enabled in a certain launch context, like a specific task type or project-wide.
+
+## Applications Addon Settings
+### Show only available applications
+![](assets/applications/show_only_available_apps.png)
+
+When enabled, AYON Launcher will display only those applications for which executable paths exist on the current machine.
+This acts as an extra filter to the Applications filters, ensuring that inaccessible applications are excluded from view.
+
+### Applications Definitions
+
+In this section you can manage what Applications are available to your studio, locations of their executables, and their additional environments. 
+In AYON context, each application that is integrated is also called a `Host` and these two terms will be used interchangeably in the documentation.
+
+#### Built-in Applications
+Each application definition has an application group e.g. Maya, Houdini, Nuke and etc.
+
+![settings_applications](assets/applications/application_grorup.png)
+
+Each Application is made of two levels.
+1. **Application group:**
+   1. **Host name:** Pipeline integration name supplied by an AYON addon.
+   2. **Environment:** Define extra environments that are applicable to all versions of the given application version dependent.
+2. **Application variant:** For each application variant we have
+   1. **Name:** application variant name
+   2. **Label:** application variant label
+   3. **Executables:** Define executables (per platform).
+   ![](assets/applications/mutliplatform_executables.png)
+   4. **Arguments:** Define default arguments (`--nukex` for instance)
+   ![](assets/applications/mutliplatform_args.png)
+   5. **Environment:** Extend application's main environment.
+   6. **+ button:** To add more variants.
+
+:::info Allowed characters in names and labels
+<!-- TODO: Note about allowed characters in host and variant names and label.
+We may mention if these name are used in other places and mostly, they are hardcoded across the pipeline because it's not expected to change them. which may bring another question about why is it an editable field.
+Also, Mention if the allowed characters may/might change in the future or not. -->
+:::
+
+#### Additional Applications
+
+It's possible to use applications addon to launch any applications that are not in built-in applications.
+However, they you won't find any AYON menus as that requires an AYON addon for that application.
```

Comment:
```suggestion
It's possible to use applications addon to launch any applications that are not in built-in applications.
However, they have any AYON integration (e.g. menus, creators, etc.) as that requires an AYON addon for that application.
```

I wonder if it's worth it to put the "They won't have any active AYON integration after launch" message in a note?
```suggestion
It's possible to use applications addon to launch any applications that are not in built-in integrations that come with AYON.

:::note
Additional applications will only _launch_ but offer no further AYON integration (e.g. menus, creators, publishers) as that requires an AYON addon for that application, at which point it should not be under _additional applications_ anymore.
:::
```